security/security-spring/src/main/java/org/cbioportal/security/spring/authentication/saml/MSKCCPortalUserDetailsService.java

```diff
@@ -30,14 +30,16 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package org.mskcc.cbio.portal.authentication.saml;
+package org.cbioportal.security.spring.authentication.saml;
```

Comment:
When we get around to merging @pieterlukasse SAML updates, I think we should remove this and the restful authentication handler from the codebase.  We can discuss more when you get to it.